airbyte-webapp/scripts/load-declarative-schema.sh

```diff
@@ -0,0 +1,26 @@
+set -e
+mkdir -p build
+
+# Make sure this is aligned with the CDK version of the connector builder server
+DEFAULT_CDK_VERSION="0.25.0"
```

Comment:
Just confirming, is the plan to address keeping these aligned automatically in a follow-up PR, since in the description you said this PR does not fully close that issue?

Do you have an idea of how to solve that?